Classes/YLRefreshHeaderView/YLRefreshHeaderView.m

```diff
@@ -61,7 +61,7 @@ - (CGFloat)pullAmountToRefresh {
 
 #pragma mark UIScrollViewDelegate
 
-- (void)containingScrollViewDidScroll:(UIScrollView *)scrollView {
+- (void)containingScrollViewDidScroll:(nonnull UIScrollView *)scrollView {
```

Comment:
`nonnull` here is implicit as it's defined in `YLRefreshHeaderViewPrivate.h`. Are we choosing to also use the nullability identifier in the implementation files for documentation?